README.md

```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+# hw01
+by Tina Chen
+
+### Biography
+Hi! my name is *Peiying Chen*, but I go by **Tina**. I am a first year in the college double majoring in **economics and psychology**. I came from China, and I moved to Irvine, California when I was in middle school. 
+
+During my free time, I enjoy doing: 
+
+* exercising 
+  * yoga
+  * swimming 
+* embroidery
```

Comment:
I would suggest putting a picture of your embroidery art in if you wanted to, that could personalize the bio even more!